core/templates/pages/story-viewer-page/story-viewer-page.directive.html

```diff
@@ -1,255 +1,90 @@
+                  <p style="font-size: 30px; padding-top: 3px;" class="oppia-chapter-title" aria-live="assertive">
+                    Chapter {{$index + 1}}: <[node.getTitle()]>
+                  </p>
 
-        <div class="oppia-card-preview-panel d-none d-md-block">
-          <md-card class="oppia-activity-summary-tile md-default-theme">
-            <div class="title-section" style="background-color: <[$ctrl.thumbnailBgColor]>; z-index: 1;">
-              <img ng-src="<[$ctrl.getStaticImageUrl($ctrl.thumbnailIconUrl)]>" class="thumbnail-image">
-            </div>
-            <div class="title-section-mask objective oppia-activity-summary-tile-pre-hover-preview">
-              <span>Hover over an icon to preview an exploration.</span>
+                  <div style="display: flex; margin-top: 16px; flex-wrap: wrap;">
+                    <svg width="240"
+                         height="180"
+                         xmlns="http://www.w3.org/1600/svg"
+                         xmlns:xlink="http://www.w3.org/1999/xlink"
+                         version="1.1"
+                         style="position:relative; margin-right: 10px; margin-bottom: 10px;">
+                      <defs>
+                        <pattern id="image<[$index]>" patternUnits="userSpaceOnUse" height="160" width="240">
+                          <rect width="240"
+                                height="180"
+                                ng-attr-fill="<[$ctrl.pathIconParameters[$index].thumbnailBgColor]>"/>
+
+                          <image x="0"
+                                 y="0"
+                                 height="180"
+                                 width="240"
+                                 ng-attr-xlink:href="<[$ctrl.getStaticImageUrl($ctrl.pathIconParameters[$index].thumbnailIconUrl)]>"
+                                 xlink:href="">
+                          </image>
+                        </pattern>
+                      </defs>
+                      <a xlink:href="<[$ctrl.getExplorationUrl(node)]>">
+                        <circle ng-show="$ctrl.node.getExplorationId() === $ctrl.nextExplorationId &&
+                          $index !== $ctrl.activeHighlightedIconIndex"
+                                cx="50"
+                                cy="70"
+                                r="50"
+                                id="<['highlight' + $index]>"
+                                fill="#FF8C00"
+                                fill-opacity="0.5"/>
+                        <rect width="240"
+                              height="180"
+                              fill="url(#image<[$index]>)"
+                              rx="8" />
+                      </a>
+                    </svg>
+
+                    <div style="min-width: 200px; max-height: 180px; overflow-y: scroll; display: flex; flex: 1; padding: 20px; border: rgb(248, 248, 248) 4px solid; border-radius: 8px; margin-bottom: 20px;">
```

Comment:
Avoid using inline styles. Use classnames instead.

Please address this here and in other place in your PR.

Reply:
Done.